src/sensai/vector_model.py

```diff
@@ -189,13 +191,19 @@ def __init__(self, checkInputColumns=True):
             e.g. in ensemble models.
         """
         super().__init__()
-        self._isFitted = False
+        self._modelIsFitted = False
         self._predictedVariableNames = None
         self._modelInputVariableNames = None
         self._modelOutputVariableNames = ["UNKNOWN"]
         self._targetTransformer: Optional[InvertibleDataFrameTransformer] = None
         self.checkInputColumns = checkInputColumns
 
+    # for backwards compatibility with persisted models based on code prior to commit 7088cbbe
+    def __setstate__(self, d):
```

Comment:
Is this really necessary just for the property rename? Since the property is private, I don't think so.

Reply:
Yeah, not worth the additional complexity. I Reverted the change in name it and added a block comment to the _isFitted instead.